phpunit.xml.dist

```diff
@@ -9,17 +9,20 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="vendor/symfony-cmf/testing/bootstrap/bootstrap.php"
 >
     <php>
-        <!-- Disable E_USER_DEPRECATED until 3.0 -->
-        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
```

Comment:
why do you remove the doc for the magic setting of error_reporting? we should keep that